src/db/migrate.rs

```diff
@@ -438,6 +438,25 @@ pub fn migrate(version: Option<Version>, conn: &mut Client) -> CratesfyiResult<(
             ALTER TABLE owners ALTER COLUMN name DROP NOT NULL;
             ALTER TABLE crates ALTER COLUMN github_stars DROP NOT NULL;
             "
+        ),
+        migration!(
+            context,
+            // version
+            18,
+            // description
+            "Add more information into doc coverage",
+            // upgrade query
+            "
+            ALTER TABLE doc_coverage
+                ADD COLUMN total_items_needing_examples INT,
+                ADD COLUMN items_with_examples INT;
```

Comment:
```suggestion
                ADD COLUMN total_items_needing_examples INT NOT NULL,
                ADD COLUMN items_with_examples INT NOT NULL;
```

Reply:
Won't it break for existing content?

Reply:
Oh you're right, there are existing docs without this info. Yeah, this needs to be nullable then.